readme.md

```diff
@@ -0,0 +1,40 @@
+﻿# SC Duty List Web Page
+
+The SC (Student Councillor for short) Duty List Web Page is for Student Councillors to use to easily check for their duty spots. 
```

Comment:
Codacy has a fix for the issue: [Expected: 0 or 2; Actual: 1](https://app.codacy.com/gh/Davianlols6/SC-Duty-List/pullRequest?prid=8390593)
```suggestion
The SC (Student Councillor for short) Duty List Web Page is for Student Councillors to use to easily check for their duty spots.
```